app/bundles/LeadBundle/EventListener/LeadSubscriber.php

```diff
@@ -529,18 +530,32 @@ protected function addTimelineDoNotContactEntries(Events\LeadTimelineEvent $even
 
                 if (!empty($row['channel'])) {
                     if ($channelModel = $this->getChannelModel($row['channel'])) {
+                        if ($channelModel instanceof ChannelTimelineInterface) {
```

Comment:
Those interfaces are meant to be used on models for 3rd party - but aren't used actively by anything in Mautic.